docker/dockerfile

```
@@ -46,4 +46,6 @@ RUN git clone https://github.com/gitleaks/gitleaks.git \
   && make build
 ENV PATH="${PATH}:/gitleaks"
 
+RUN sudo curl -o /usr/local/bin/jq -L https://github.com/stedolan/jq/releases/download/jq-1.8.1/jq-linux64 && sudo chmod +x /usr/local/bin/jq
```

Hadolint (2.14.0)

[error] Do not use sudo as it leads to unpredictable behavior. Use a tool like gosu to enforce root

(DL3004)